projects/components/src/filtering/filter/filter-attribute.ts

```diff
@@ -7,4 +7,5 @@ export interface FilterAttribute {
   type: FilterAttributeType;
   onlySupportsAggregation?: boolean;
   onlySupportsGrouping?: boolean;
+  scope?: string;
```

Comment:
Do we need this `scope` entry here given we have created a type called `SavedQuery` which contains both `scope` and `filter`?

Reply:
The `scope` entry here is for the `metadata` property which is of type `FilterAttribute`:
https://github.com/razorpay/hypertrace-ui/pull/32/files#diff-f19fe761c23b87fda9dc84db21cb18ead475913118bdff3f4f2e7af39990754bR205

Added the entry to avoid the typescript error saying: "scope doesn't exist on metadata"

Reply:
I replaced `metadata.scope` with `currentContext`. This type modification is not needed anymore. Removed ✅